csharp/lang/security/memory/memory-marshal-create-span.yaml

```diff
@@ -0,0 +1,22 @@
+rules:
+  - id: memory-marshal-create-span
+    severity: WARNING
+    languages:
+      - C#
+    metadata:
+      cwe:
+        - "CWE-125: Out-of-bounds Read"
```

Comment:
This is OK, but a warning is that I think up 'til now CWE has always been a string. I think this will cause issues in the app

Reply:
Just based on previous experience, I think it will cause issues much like the OWASP update. I don't currently see any other list-based CWEs. I think this highlights the need for a metadata schema for rules in this repository. Will talk to some others about this